app.js

```diff
@@ -13,324 +13,287 @@ if (!supportsLocalStorage()) {
+     if (invitee === current.name.replace(/\s/g, '')) {
+       current.attending = false;
+       current.notAttending = true;
+     }
+   });
+   renderedInviteList.map((current, index) => {
+     if (invitee === current.name.replace(/\s/g, '')) {
+       current.attending = false;
+       current.notAttending = true;
+     }
+   });
+   refreshLocalStorageList(fullInviteList);
+  };
+
+  //re-maps renderedInviteList to include only attending respondants. Or to include all invitees, if filter is unchecked
+  const filterAttending = (eventTarget) => {
+    renderedInviteList = [];
+    if (eventTarget.checked) {
+      fullInviteList.map((current, index) => {
+    	if (current.attending === true) {
+        renderedInviteList.push(current);
       }
+      });
+    } else if (!eventTarget.checked) {
+      fullInviteList.map((current, index) => {
+        renderedInviteList.push(current);
+      });
     }
-  }
+   reloadDOM();
+  };
 
-  //Hide the selected li label when corresponding filter is applied
-  function hideSelectedLabel() {
-    const lis = ul.children;
-    for (i = 0 ; i < lis.length ; i++) {
-      let li = lis[i];
-      let attending = li.firstElementChild.nextElementSibling.firstElementChild;
-      let attendingParent = attending.parentNode;
-      let nextLabel = attendingParent.nextElementSibling;
-      if (attending.checked === true) {
-        attendingParent.style.display = 'none';
-        nextLabel.style.display = '';
-      } else if (nextLabel.firstElementChild.checked === true) {
-        nextLabel.style.display = 'none';
-        attendingParent.style.display = '';
+  //re-maps renderedInviteList to include only not attending respondants. Or to include all invitees, if filter is unchecked
+  const filterNotAttending = (eventTarget) => {
+  renderedInviteList = [];
+  if (eventTarget.checked) {
+    fullInviteList.map((current, index) => {
+      if (current.notAttending === true) {
+        renderedInviteList.push(current);
       }
-    }
+    });
+  } else if (!eventTarget.checked) {
+    fullInviteList.map((current, index) => {
+      renderedInviteList.push(current);
+    });
   }
+  reloadDOM();
+  };
 
-  //Show all li labels
-  function showAllLabels() {
-    const lis = ul.children;
-    for (i = 0 ; i < lis.length ; i++) {
-      let li = lis[i];
-      let attending = li.firstElementChild.nextElementSibling.firstElementChild;
-      let attendingParent = attending.parentNode;
-      let nextLabel = attendingParent.nextElementSibling;
-      attendingParent.style.display = '';
-      nextLabel.style.display = '';
-    }
+  //re-maps renderedInviteList to include invitees who have not responded. Or to include all invitees, if filter is unchecked
+  const filterTheWorstKindOfPeople = (eventTarget) => {
+  renderedInviteList = [];
+  if (eventTarget.checked) {
+    fullInviteList.map((current, index) => {
+      if ((current.attending === false) && (current.notAttending === false)) {
```

Comment:
Could you just use a logical NOT operator here?
```
if (!current.attending && !current.notAttending) {...
```

Reply:
At least I'm pretty?